app/build.gradle

```diff
@@ -23,7 +25,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
         debug {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Good catch, that was required back when I supported Android KitKat since it had too many classes and that didn't work well with Dalvik

Reply:
Thanks to new AndroidX c:
I'm testing the app now, because did some shit with github XD